data/base/endpoint.php

```diff
@@ -104,28 +130,24 @@ protected function register() {
 	 * @param string $route
 	 * @param string $endpoint_class
 	 *
-	 * @return \Elementor\Data\Base\SubEndpoint
-	 * @throws \Exception
+	 * @return \Elementor\Data\Base\SubEndpoint|false
 	 */
-	protected function register_sub_endpoint( $route, $endpoint_class ) {
-		$endpoint_instance = new $endpoint_class( $route, $this );
+	public function register_sub_endpoint( $route, $endpoint_class ) {
```

Comment:
Try use the same function from parent.

Reply:
Its not possible since register_sub_endpoint store sub endpoints in endpoint class.